### ISSUE TITLE
Can't install on VS 2019

### ISSUE THREAD

**Issue by gagejd** [atlaste/CPPCoverage#25]
Can't install on Visual Studio 2019.  Here is the log:

[dd_VSIXInstaller_20190620132045_467c.log](https://github.com/atlaste/CPPCoverage/files/3311452/dd_VSIXInstaller_20190620132045_467c.log)

**Comment by atlaste** [comment]
Yeah it's not built for VS2019 yet, simply because we have no access to an installed version of VS2019. Only thing you need installed is the VS SDK, which ships with VS by default.

In other words, you can just compile it and it should work just fine.

**Comment by gagejd** [comment]
Thanks Stefan!

I'll give that a go.

John

**Comment by atlaste** [comment]
Let me know if everything worked out please; you're probably not the only person who wants this.

**Comment by atlaste** [comment]
I just made some changes to the version prerequisites of the installer. The irony is that there's really no reason why it shouldn't install... Anyways, if you haven't built it already, could you try again?

**Comment by gagejd** [comment]
Thanks Stefan.  That seemed to do the trick.I will put it into regular use
and get back to you with any comments.

John

**Comment by gagejd** [comment]
Just curious...

I ran the coverage and can see the source code highlighted by color.  That
worked great.  Now I'm back to work on the code and don't need the coloring
anymore. Is there a way to turn off the coloring until the next coverage
run?

John

**Comment by atlaste** [comment]
Yeah, make some change in your code. Or just remove the coverage file.

**Comment by Rominitch** [comment]
Hi Atlaste,
I make some job too about new system for VisualStudio 2019 extension. Now, we need to deferred loading (otherwise extensions' manager need to activate sync mode).

Don't hesitate to look my fork and take useful code.

Regards

**Comment by atlaste** [comment]
@PropelledDog Sorry for the late reply, I totally missed this comment.

Ideally, I'd use Github CI to just generate them. But I have no idea how to set that up. If you do, please send me a message on discord or whatever so we can set something up. (I'll put a discord link on the main page)

**Comment by atlaste** [comment]
Well I gave it my best and made some releases (manually). If this can be triggered with CI, feel free to drop a message.
